api-client/src/test/java/com/desk/java/apiclient/DeskClientTest.java

```diff
@@ -26,14 +26,21 @@
 
 package com.desk.java.apiclient;
 
+import com.desk.java.apiclient.util.DeskClientUtils;
```

Comment:
Ahh, then maybe there should be a `DeskClientUtilsTest` in which those test methods are moved to?